.circleci/config.yml

```diff
@@ -17,8 +17,8 @@ jobs:
           name: Test
           command: make test
       - run:
-          name: Integration tests # test cross build too!
-          command: make -e TARGET_TEST_ARCH=* integration_test
+          name: Integration tests
```

Comment:
Dropped cross-qemu build during integration tests to free up CI space/time for new builder images.